ui/tests/__tests__/App.test.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import App from "../../src/App";
+import { mockFacetsGet } from "../../src/api/src/api/FacetsApi";
+import { mockDatasetGet } from "../../src/api/src/api/DatasetApi";
+
+jest.mock("../../src/api/src/api/DatasetApi");
+jest.mock("../../src/api/src/api/FacetsApi");
+("use strict");
+
+beforeEach(() => {
```

Comment:
As discussed, please delete toHaveBeenCalledTimes() in all tests, except for "Stringifies queries when calling the API to update Facets".

Can beforeEach() be deleted?